beacon-chain/blockchain/core.go

```diff
@@ -242,6 +242,16 @@ func (b *BeaconChain) computeNewActiveState(seed common.Hash) (*types.ActiveStat
 	}, nil
 }
 
+// getAttestersProposer returns lists of random sampled attesters and proposer indices.
+func (b *BeaconChain) getAttestersProposer(seed common.Hash) ([]int, int, error) {
+	attesterCount := math.Min(params.AttesterCount, float64(len(b.CrystallizedState().ActiveValidators)))
```

Comment:
Is this correct?. `params.AttesterCount` is the number of attesters per committee, not the max # of validators allowed.

Reply:
Doesn't look right. `getAttestersProposer` was implemented before this PR, and looks like it's going away in spec 2.1 
I'll fix it!